app/views/user_mentee_applications/show.html.erb

```diff
@@ -1,70 +1,94 @@
-<div class="max-w-prose mx-auto p-4">
-  <section class="flex flex-col mb-3 mb-5 ">
-    <div class="mx-auto text-2xl text-red-500 mb-3">
-      <%= @user_mentee_application.current_status.to_s.humanize %>
+<div class="mx-auto p-4 my-8 max-w-prose">
+  <section class="flex flex-col my-4 mx-auto rounded-md border-2">
+    <div class="inline my-4 mx-auto pt-4">
+      <span class="text-md text-base-content/70">Application Status</span>
```

Comment:
Status needs a label, it's not data that is self-explanatory enough for an applicant that the data can be freestanding.